app/assets/stylesheets/core-layout.scss

```diff
@@ -4,6 +4,4 @@
 // import the base styles
 @import "header-footer-only";
 
-// import layouts
-@import "helpers/layouts";
```

Comment:
Can we merge this with the first commit where we remove `govuk-grid` https://github.com/alphagov/static/pull/2014/commits/45c92d133e7c23c4af9f80cb50ac0527ab525959?

Reply:
I'm working on reordering and merging a lot of these commits to make this change easier to read.